tests/containers/integration-test-base

```diff
@@ -8,3 +8,9 @@ RUN dnf upgrade --refresh -y --nodocs && \
         systemd-devel \
     -y && \
     dnf -y clean all
+
+# First, install dasbus via dnf (v1.4) so the package can be provided.
+# Then upgrade dasbus via pip to v1.7 (required by python3-bluechi)
+RUN dnf install -y python3-dasbus
```

Comment:
If I have to install `dasbus` via `pip` why do I need to first install it with `dnf`? Installing the same content via two different package managers can lead to mismatches.

Reply:
When only installing it via `pip`, the subsequent `dnf install python3-bluechi` doesn't recognize that its installed. However, `dnf install python3-dasbus` only has v1.4 while the bluechi bindings need v1.7. This is definitely not an ideal solution. Is there a way to better solve this? Maybe dropping the requirement in the rpm spec and just install via `pip`?

Reply:
I think that for now it should be dropped from the dependencies listed in the `python3-bluechi` RPM and installed only via pip (and documenting it).
Having said that, as @mwperina wrote:
> we should work with RHEL maintainers to try to do a version bump in CS9

Reply:
As discussed in the sync yesterday, I was able to have our `python3-bluechi` bindings compatible with both versions - v1.4 and v1.7 (at least I tested with those, the versions in between I didn't verify). The root cause was the data type `File` (in v1.4+) had been renamed to `UnixFD` (in [v1.7](https://github.com/rhinstaller/dasbus/releases/tag/v1.7)) in [this PR](https://github.com/rhinstaller/dasbus/pull/70/files). Adding a try-wrapped import solves the issue. 

I noticed slightly different behavior between the versions, e.g. when [shutting down the agent service](https://github.com/containers/bluechi/pull/485/files#diff-57474d9f936f33187853fdba978ee5de1daadffde2fccfa2e25624dc8828c3d1R27-R37). With v1.4 the `stop_unit` raises an Error for connection timeout while v1.7 doesn't. The main goal (disconnecting the agent) is still achieved, so its ok-ish. I'll have brief look if I can improve this nonetheless.